src/main/java/technology/tabula/extractors/BatchSelectionExtractor.java

```
@@ -0,0 +1,473 @@
+											stringList, coordList, pageList, tables, tableHeaders);
+
+									pdfDocument.close();
+
+									reading = false;
+
+									// add document to list of created files instead? clean up later?
+									// can this trigger an exception too?
+									// need another try-catch?
+								} catch (Exception e) {
+									// e.printStackTrace();
+									System.out.println("Unable to properly convert or extract data from document");
+								}
+
+							} else
+								System.out.println("Ignored document based on input parameters");
+						}
+
+						// just in case
+						if (reading) {
+							pdfDocument.close();
+							reading = false;
+						}
+						// write data to new file
+						// currently overwrites existing files and creates a
+						// 	file even if no tables extracted
+						// may be better to check if file already exists
+						Writer writer = null;
+						BufferedWriter bufferedWriter = null;
+						File testFile = new File(
+								outputPath + "\\" + fileName.substring(0, fileName.lastIndexOf(".")) + ".csv");
+						testFile.createNewFile();
+						FileWriter fileWriter = new FileWriter(testFile);
+						bufferedWriter = new BufferedWriter(fileWriter);
+						// assuming CSV format for demo
+						writer = new CSVWriter();
+
+						int tableNum = 0; // needed?
+
+						// add header for OCR'd files?
+						for (Table table : tables) {
+							List<Table> fauxTableList = new ArrayList<Table>();
+							fauxTableList.add(table);
+
+							// write table header
+							bufferedWriter.write(tableHeaders.get(tableNum) + "\n");
+
+							// write single table
+							writer.write(bufferedWriter, fauxTableList);
+
+							// write line break
+							bufferedWriter.write("\n");
+
+							// increment table number?
+							tableNum++;
+						}
+
+						// shut the door on your way out
+						bufferedWriter.close();
+						// pdfDocument.close();
+					}
+
+					catch (Exception e) {
+						e.printStackTrace();
```

If swallowing all possible Exceptions, please explain why. This is likely incorrect. Also, do not print stack trace to console.